examples/evaluation/lora-evaluation.ipynb

```diff
@@ -0,0 +1,4369 @@
+      "Downloading tabledata-1.3.3-py3-none-any.whl (11 kB)\n",
+      "Downloading tcolorpy-0.1.6-py3-none-any.whl (8.1 kB)\n",
+      "Downloading typepy-1.3.2-py3-none-any.whl (31 kB)\n",
+      "Downloading portalocker-2.10.1-py3-none-any.whl (18 kB)\n",
+      "Downloading chardet-5.2.0-py3-none-any.whl (199 kB)\n",
+      "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m199.4/199.4 kB\u001b[0m \u001b[31m15.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
+      "\u001b[?25hBuilding wheels for collected packages: lm_eval, rouge-score, sqlitedict, word2number\n",
+      "  Building wheel for lm_eval (pyproject.toml) ... \u001b[?25ldone\n",
+      "\u001b[?25h  Created wheel for lm_eval: filename=lm_eval-0.4.5-py3-none-any.whl size=2385487 sha256=25fa85f7ef4fe414f5f7afc73ebba25257ff7897659cb1c81bc9c27968838102\n",
+      "  Stored in directory: /tmp/pip-ephem-wheel-cache-qz890orw/wheels/ff/d5/f4/949f2b285369fc332f8dbba43f220ebd88387ee2f2c0c1174f\n",
+      "  Building wheel for rouge-score (setup.py) ... \u001b[?25ldone\n",
+      "\u001b[?25h  Created wheel for rouge-score: filename=rouge_score-0.1.2-py3-none-any.whl size=24934 sha256=aef8d4252a9435d1b409aef8bd28b57e731292afd04044dc0c26c4e093ea5ea0\n",
+      "  Stored in directory: /root/.cache/pip/wheels/5f/dd/89/461065a73be61a532ff8599a28e9beef17985c9e9c31e541b4\n",
+      "  Building wheel for sqlitedict (setup.py) ... \u001b[?25ldone\n",
+      "\u001b[?25h  Created wheel for sqlitedict: filename=sqlitedict-2.1.0-py3-none-any.whl size=16862 sha256=46fa3bc0d445b7d8e84c8d668b053dd24125ebbe4bfe1eb50261b8f0c2c82418\n",
+      "  Stored in directory: /root/.cache/pip/wheels/79/d6/e7/304e0e6cb2221022c26d8161f7c23cd4f259a9e41e8bbcfabd\n",
+      "  Building wheel for word2number (setup.py) ... \u001b[?25ldone\n",
+      "\u001b[?25h  Created wheel for word2number: filename=word2number-1.1-py3-none-any.whl size=5566 sha256=274e019bfbf4a3b12c926cd53d4d31e0fe240e59794b9370dbf994833afb50c2\n",
+      "  Stored in directory: /root/.cache/pip/wheels/84/ff/26/d3cfbd971e96c5aa3737ecfced81628830d7359b55fbb8ca3b\n",
+      "Successfully built lm_eval rouge-score sqlitedict word2number\n",
+      "Installing collected packages: word2number, sqlitedict, tqdm-multiprocess, tcolorpy, portalocker, pathvalidate, jsonlines, chardet, sacrebleu, rouge-score, mbstrdecoder, typepy, peft, evaluate, DataProperty, tabledata, pytablewriter, lm_eval\n",
+      "Successfully installed DataProperty-1.0.1 chardet-5.2.0 evaluate-0.4.3 jsonlines-4.0.0 lm_eval-0.4.5 mbstrdecoder-1.1.3 pathvalidate-3.2.1 peft-0.13.2 portalocker-2.10.1 pytablewriter-1.2.0 rouge-score-0.1.2 sacrebleu-2.4.3 sqlitedict-2.1.0 tabledata-1.3.3 tcolorpy-0.1.6 tqdm-multiprocess-0.0.11 typepy-1.3.2 word2number-1.1\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Install LM-Eval\n",
+    "!pip install git+https://github.com/EleutherAI/lm-evaluation-harness.git"
```

Comment:
Can we not install the normal pypi package? Let's only use a source install if the pypi package lacks a feature we need.